platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/migration/SystemConfigurationMigrationTest.java

```diff
@@ -143,19 +145,42 @@ public void setup() throws Exception {
     }
 
     @Test
+    public void testSymbolicLinkPath() throws Exception {
+
+        System.setProperty(KEYSTORE_SYSTEM_PROP, KEYSTORE_REL_PATH);
+        System.setProperty(TRUSTSTORE_SYSTEM_PROP, TRUSTSTORE_REL_PATH);
+        SystemConfigurationMigration securityConfigurationMigrator =
+                new SystemConfigurationMigration(createTempDdf()) {
+
+                    @Override
+                    Properties readPropertiesFile(Path propertiesFile) throws MigrationException {
+                        Properties properties = new Properties();
+                        properties.setProperty(CRL_PROP_KEY, symLinkPath.toString());
+                        return properties;
+                    }
+                };
+
+        // Perform Test
+        assertFalse("A migration warning wasn't returned.",
```

Comment:
We should be able to replace this assertion (and all the other `assertFalse()` ones in this test file) with `assertThat("A migration warning wasn't returned.", securityConfigurationMigrator.export(...), is(not(empty())))`.